internal/statestore/ticket_test.go

```diff
@@ -903,6 +903,7 @@ func createRedis(t *testing.T, withSentinel bool, withPassword string) (config.V
 	cfg.Set("redis.pool.maxActive", 5)
 	cfg.Set("backfillLockTimeout", "1m")
 	cfg.Set("pendingReleaseTimeout", "200ms")
+	cfg.Set("backfillTimeToLive", "800ms")
```

Comment:
@hsorellana I thought you would change this `800ms` time to some per cent value and rename this `backfillTimeToLive` to something like `backfillTTL` .